integration-tests/src/chains/pop_network/genesis.rs

```diff
@@ -1,43 +1,43 @@
 use emulated_integration_tests_common::{accounts, build_genesis_storage, collators};
-use pop_runtime::Balance;
+use pop_runtime_common::Balance;
 use sp_core::storage::Storage;
 
-pub(crate) const ED: Balance = pop_runtime::EXISTENTIAL_DEPOSIT;
+pub(crate) const ED: Balance = pop_runtime_devnet::EXISTENTIAL_DEPOSIT;
```

Comment:
We should ideally alias `pop_runtime_devnet` to `runtime` to simplify the rest of the code in the file. Something like `use pop_runtime_devnet as runtime`

Reply:
This is good idea

Reply:
Addressed in a PR to this one.